src/app/routes/applications/new-staking-pool/new-staking-pool.component.ts

```diff
@@ -17,8 +18,8 @@ const {parseEther} = utils;
   templateUrl: './new-staking-pool.component.html',
   styleUrls: ['./new-staking-pool.component.scss']
 })
-export class NewStakingPoolComponent implements OnInit {
-
+export class NewStakingPoolComponent implements OnInit, OnDestroy {
+  editor: Editor;
```

Comment:
Please add handling returned value of this editor inside form - it is supported as far as i see from docs. That way we can add validators if that will be needed and we will have only one check for enabling/disabling submit button.